extra/helpers.py

```diff
@@ -1,6 +1,15 @@
 import multiprocessing, subprocess
 import cloudpickle  # type: ignore
 from typing import Any
+from tinygrad.nn.state import get_state_dict
+
+def get_net_size(model):
+  total_params, total_bytes = 0, 0
+  for param_name, param in zip(get_state_dict(model), get_state_dict(model).values()):
+    if not ("num_batches_tracked" in param_name) and not ("running" in param_name):
```

Comment:
Err, I don't like this gate based on name.

Also, huh?!?
`zip(get_state_dict(model), get_state_dict(model).values()):`

Isn't that just:
`get_state_dict(model).items()`

Reply:
you are right. thank you for catching this.